prototype/northbound/sensorthings/rest.api/src/main/java/org/eclipse/sensinact/sensorthings/sensing/rest/filters/PropFilterImpl.java

```diff
@@ -38,6 +38,12 @@ public void aroundWriteTo(WriterInterceptorContext context) throws IOException,
         Object entity = context.getEntity();
 
         String propName = uriInfo.getPathParameters().getFirst("prop");
+        if(propName == null) {
+            // No property filter
```

Comment:
Isn’t this an error? The prop filter name binding is supposed to be used with a path parameter that cannot be null, so I’m not sure how this code is reached. If it is reached we should abort with a failure.

Reply:
Agreed. This case only happens if the annotated method doesn't have a "prop" parameter.